go/web/skeleton/Makefile

```diff
@@ -1,7 +1,7 @@
 projectDir := $(realpath $(dir $(firstword $(MAKEFILE_LIST))))
 os := $(shell uname)
-app_name = go-web
-tenant_name = $(TENANT_NAME)
+app_name = {{ name | default("go-web") }}
+tenant_name = {{ tenant | default("$(TENANT_NAME)") }}
```

Comment:
do we have docs saying that we can set that variable and the tenant will be populated?

Reply:
`corectl app create -h` mentions it, but will add some more documentation to docs once I'm done